daemon/cmd/daemon_main.go

```diff
@@ -980,6 +980,9 @@ func init() {
 	flags.Bool(option.ExternalClusterIPName, false, "Enable external access to ClusterIP services (default false)")
 	option.BindEnv(option.ExternalClusterIPName)
 
+	flags.IntSlice(option.BPFAllowVLAN, []int{}, "List of explicitly allowed VLAN IDs, '0' id will allow all VLAN IDs")
```

Comment:
I think, no, according to talks on previous pull request. Today's behaviour doesn't mean 'allow all'. Today's behaviour is broken - cilium treats any vlan tagged packets as packets without tag and may take packet from main interface and resend to another node (for example) from vlan tagged interface and this will add one more vlan tag -> packet processing will be broken. Current behaviour does not work at all with vlan tagged packets.

Reply:
OK, in that case if it's pretty broken today then it'd be better to drop early here with clear drop notifications so if someone configures this way, they at least get a more obvious failure they can investigate rather than just weird packets turning up in unexpected places. :+1: